.github/workflows/github.yml

```diff
@@ -312,6 +375,92 @@ jobs:
         DEPLOY_RSA: ${{ secrets.DEPLOY_RSA }}
         PACKAGE_EXTENSION: ${{ matrix.extension }}
 
+    - name: Prepare partial JSON with build informations
+      id: make_partial_json
+      shell: bash
+      run: |
+        # Decide cache tool and family
+        if [[ "${{ startsWith(matrix.platform, 'msvc') }}" == "true" ]]; then
+          TOOL="sccache";  CMD="sccache --show-stats"; FAMILY="windows-msvc"
```

Comment:
why throwing everything onto a single line?

Reply:
Currently it's not used in report. I have only keep it. it's single line in json with \n. sccache and ccache have different outputs. So easier was to put it into single line. Maybe replace \n with <br> will be better later when adding detailed output to Summayr build report.

Reply:
Done